src/main/java/org/spongepowered/mod/mixin/entity/player/MixinEntityPlayer.java

```diff
@@ -71,4 +75,8 @@ public MixinEntityPlayer(World worldIn) {
         return this.openContainer == this.inventoryContainer;
     }
 
+    public String human$getName() {
+        return this.getName();
```

Comment:
I'm about to push code that changes it - I'm just debugging some other issues first.